src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -809,37 +809,25 @@ private class FactoryContext
             public List<(ParameterExpression, Expression)> TryParseParams { get; } = new();
         }
 
-        private static class Log
+        internal static partial class Log
```

Comment:
Why `internal`?

Reply:
I can revert it. I was fiddling around with making the source generator work.